com.google.AndroidStudio.json

```diff
@@ -0,0 +1,77 @@
+{
+    "app-id": "com.google.AndroidStudio",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "1.6",
+    "sdk": "org.freedesktop.Sdk",
+    "command": "android",
+    "finish-args": [
+        "--socket=x11",
+        "--socket=pulseaudio",
+        "--socket=system-bus",
```

Comment:
I know it effectively doesn't matter but system bus access is probably unnecessary.